libtenzir/include/tenzir/concept/parseable/tenzir/data.hpp

```diff
@@ -44,6 +44,18 @@ constexpr inline auto null = null_parser{};
 
 } // namespace parsers
 
+struct simple_data_parser : parser_base<simple_data_parser> {
+  using attribute = data;
+
+  template <class Iterator, class Attribute>
+  bool parse(Iterator& f, const Iterator& l, Attribute& a) const {
+    static auto p = parsers::time | parsers::duration | parsers::net
+                    | parsers::ip | parsers::number | parsers::boolean
+                    | parsers::qqstr | parsers::null;
```

Comment:
Do we want `qqstr` in this?

Reply:
On that note, do we want `null` there? If it's there, there's no way to differentiate between a non-match and a matched string "null".

Reply:
Both removed in latest commit.